qvm/anon-whonix.sls

```diff
@@ -21,12 +21,13 @@ include:
   - qvm.sys-whonix
   - qvm.whonix-ws-dvm
 
+{%- from "qvm/whonix.jinja" import whonix with context -%}
```

Comment:
Why `with context`?

Reply:
No particular reason - I don't have much personal experience doing this, and most examples I've seen use it that way, I guess allows for quite clear namespacing and not clobbering other variables. Would you prefer it removed?

Reply:
According to [docs](http://jinja.pocoo.org/docs/2.10/templates/#import-visibility), better avoid 'with context' because it disable caching. This file doesn't use any variable defined outside of it (I assume `salt` is available anyway).
Also, didn't you meant `import ... as whonix` instead of `from ... import whonix`? There is no `whonix` defined in that file (only `whonix_version`).

Reply:
This is sorted with the latest commit https://github.com/QubesOS/qubes-mgmt-salt-dom0-virtual-machines/pull/10/files/4dcf91e3257e83a90ea7c7e99a74f66b7f410a1b..d2bfbf3504c70b7147c1c5d6a0546ff4a5d8431f by @viq.